src/routes/__layout-root.svelte

```diff
@@ -37,7 +37,7 @@
 
     const cluster: GetClusterInfoResponse = await fetchCluster(settings, fetch);
 
-    fetchSearchAttributes(settings);
+    fetchSearchAttributes(settings, fetch);
```

Comment:
Resolves:

![image](https://user-images.githubusercontent.com/251288/192444115-0afe59be-f4cf-4241-82cd-3dad87ec3aff.png)

Reply:
If you want to break this out into it’s own PR, I’m comfortable merging this in while Candace reviews the design/copy change.

Reply:
@stevekinney  talked to Candace and removed the copy, so I think this is ready for merging ☺️